test/battle/move_effect/fury_cutter.c

```diff
@@ -36,3 +36,27 @@ SINGLE_BATTLE_TEST("Fury Cutter power doubles with each use, up to 160 power")
         EXPECT_EQ(damage[maxTurns - 2], damage[maxTurns - 1]);
     }
 }
+
+SINGLE_BATTLE_TEST("Fury Cutter counter is the same for both hits of Parental Bond")
+{
+    s16 damage[4];
+
+    GIVEN {
+        PLAYER(SPECIES_WOBBUFFET) { Ability(ABILITY_PARENTAL_BOND); }
+        OPPONENT(SPECIES_LINOONE) { HP(900); }
```

Comment:
Could you use Regirock or something similar with a high defense to avoid specifying the HP stat?